bids/variables/entities.py

```diff
@@ -80,10 +80,11 @@ class RunInfo(RunInfo_):
     pass
 
 
-class NodeIndex(Node):
+class NodeIndex(object):
     """Represents the top level in a BIDS hierarchy. """
 
     def __init__(self):
+        super().__init__()
```

Comment:
Last gasp of Python 2.

```suggestion
        super(NodeIndex, self).__init__()
```